style/components/snackbar.css

```diff
@@ -0,0 +1,40 @@
+#snackbar {
+  background: rgb(54, 54, 54);
+  color: white;
+  padding: 15px;
+  display: inline-block;
+
+  position: fixed;
+  top: 30px;
+  right: 40px;
+  transform: translateX(300px);
+  transition: all 500ms;
+  z-index: 9999;
+}
+
+#snackbar-center {
+  font-size: 80%;
+  background: rgb(54, 54, 54);
+  color: white;
+  padding: 10px 20px;
+  display: inline-block;
+
+  position: fixed;
+  bottom: 70px;
+  left: 550px;
```

Comment:
This is not good practice for a responsive point of view. Use rem instead px

Reply:
sure I will change it